AppCenter.xcworkspace/xcshareddata/xcschemes/All App Center Frameworks.xcscheme

```diff
@@ -3,7 +3,7 @@
    LastUpgradeVersion = "1150"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "YES"
+      parallelizeBuildables = "NO"
```

Comment:
I wonder why it doesn't work with parallelized builds.

Reply:
Now the App Center core should assemble everytime when we going to assemble any of our modules and if we run build all our modules in parallel, then the build will be interrupted.

Reply:
I'd try to keep fix this problem to keep it parallel